### PR TITLE
ECL-854: Update statement

### PR DESCRIPTION
**Pull request by paddynski-opencast**

### CODE REVIEW

File: conf/services/economic-crime-levy.yml [hunk 1]
```diff
@@ -1,32 +1,19 @@
-serviceName: Economic Crime Levy                # The service name that will appear in the title of the accessibility statement.
+serviceName: Economic Crime Levy                     # The service name that will appear in the title of the accessibility statement.
```
Comment (TheDanielDoyle) [inline conf/services/economic-crime-levy.yml:1]: Remove all YAML comments

File: conf/services/economic-crime-levy.cy.yml [hunk 2]
```diff
@@ -6,26 +6,13 @@ serviceDescription: |
...
-    date: 2023-10-27
-  - description: Mewnbwn ffôn ar goll ar gyfer maes rhif ffôn. Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.5 (Identify Input Purpose) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
-    date: 2023-10-27
-serviceLastTestedDate: 2023-06-29
+complianceStatus: full
+serviceLastTestedDate: 2024-04-16
 statementVisibility: public
 statementCreatedDate: 2023-08-31
-statementLastUpdatedDate: 2023-08-31
+statementLastUpdatedDate: 2024-04-17
```
Comment (TheDanielDoyle) [inline conf/services/economic-crime-levy.cy.yml:13]: Today's date please.

File: conf/services/economic-crime-levy.yml (hunk 1 above)
Comment (paddynski-opencast) [reply conf/services/economic-crime-levy.yml]: Done.

File: conf/services/economic-crime-levy.yml [hunk 3]
```diff
@@ -1,32 +1,19 @@
...
-statementCreatedDate: 2023-08-31        # In ISO format YYYY-MM-DD
-statementLastUpdatedDate: 2023-08-31    # In ISO format YYYY-MM-DD
+serviceDomain: www.tax.service.gov.uk                # The domain name under which this service exists (exclude the https:// and the path)
+serviceUrl: /register-for-economic-crime-levy        # The relative URL to the service (omitting www.tax.service.gov.uk)
+contactFrontendServiceId: economic-crime-levy        # A unique identifier for your service (see notes below)
+complianceStatus: full                               # full|partial|noncompliant
+serviceLastTestedDate: 2024-04-16                    # In ISO format YYYY-MM-DD. If your statement's compliance status is noncompliant, you can omit this line
+statementVisibility: public                          # If set to public, the statement will be visible in production. One of: public|draft|archived.
+statementCreatedDate: 2023-08-31                     # In ISO format YYYY-MM-DD
+statementLastUpdatedDate: 2024-04-17                 # In ISO format YYYY-MM-DD
```
[the remark this answers is not in the document]
Comment (paddynski-opencast) [reply conf/services/economic-crime-levy.yml]: Done.

**Review by TheDanielDoyle — APPROVED** [review]

**Pull request merged**